pkg/generate/generate.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package generate scaffolds components to guide a user for component creation
+package generate
+
+import "io/ioutil"
+import "os"
+import "path"
+import "strings"
+
+const componentBuilder = `apiVersion: bundle.gke.io/v1alpha1
+kind: ComponentBuilder
```

Comment:
Can we also add
1. an example patch
2. a test-suite test

Reply:
If you're time limited, I'd recommend doing just the patch.

Reply:
@Kashomon Yea let's get this functionality in an I'll merge that behavior in the morning if that's okay.